common/src/include/ebbrt/IOBuf.h

```diff
@@ -179,6 +211,12 @@ class IOBuf {
       return ret;
     }
 
+    // copies 'size' bytes into pointer 'vaddr'
+    void Get(size_t size, void* vaddr) {
```

Comment:
this should really be a `uint8_t *`, and you probably want to call it `buf` or something, not `vaddr`